File: app/src/main/java/com/example/vladimir/easyenglishlearn/db/AppDatabase.java
package com.example.vladimir.easyenglishlearn.db;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.example.vladimir.easyenglishlearn.model.Word;

@Database(entities = {Word.class}, version = 1, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {

    public abstract WordDao wordDao();
}

File: app/src/main/java/com/vladimir_x/easyenglishlearn/db/AppDatabase.kt
package com.vladimir_x.easyenglishlearn.db

import androidx.room.Database
import com.vladimir_x.easyenglishlearn.model.Word
import androidx.room.RoomDatabase
import com.vladimir_x.easyenglishlearn.db.WordDao

@Database(entities = [Word::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
    abstract fun wordDao(): WordDao?
}